### ISSUE TITLE
InfluxDB with duplicate records and without data

### ISSUE THREAD

**Issue by vladurash** [calmjm/tojota#26]
I found 2 issues:
1. If you clear cache folder for any reason, without clearing InfluxDB, repeating fetching data will result in multiple duplicates
2. All records are inserted in InfluxDB having current datetime of running script, not datetime of each trip (for example). I think you should include more columns or to parse all json files.
`> select * from trip_kilometers
name: trip_kilometers
time                value
----                -----
1612647596335809847 13.448
1612647597243310468 6.365
1612647597835829061 4.262
1612647598594505394 23.921
1612647599368369993 0.353
1612647600001367591 9.555
1612647600743092876 3.436
1612649311230798518 5.221
1612649312204530917 21.068
1612649313124840555 13.448
1612649313775606051 6.365
1612649314515323679 4.262
1612649315337836386 23.921
1612649315873790965 0.353
1612649316466498043 9.555
1612649317035171363 3.436
1612649685475717523 5.221
1612649686320304001 21.068
1612649687186572614 13.448
1612649687833560922 6.365
1612649688451598925 4.262
1612649689318661414 23.921
1612649689903130239 0.353
1612649690494339085 9.555
1612649691171616889 3.436
`

**Comment by calmjm** [comment]
Yes, it does that. Feel free to make PR to save timestamps from the data where available.
